text/0756-helper-default-manager.md

```diff
@@ -0,0 +1,276 @@
+such as a plain function? This RFC proposes a default behavior for
+those unknown scenarios when it comes to _helpers_.
+
+## Motivation
+
+The addon, [ember-could-get-used-to-this](https://github.com/pzuraq/ember-could-get-used-to-this)
+demonstrated that it's possible to use plain functions for helpers and modifiers.
+And since Ember 3.25, helpers can be invoked directly from value references, this opened
+a whole new world of ergonomics improvements where a dev could define a function in a
+component class and use that function **as** a helper, thanks to
+ember-could-get-used-to-this implementing a Helper Manager that _knew what to do with plain functions.
+
+This has the impact of greatly reducing helper burden for apps and addon devs, in that,
+folks no longer need to jump over to the app/addon helpers directory to create a helper
+"just to do this one simple thing". It's all now `{{this.myHelper}}` or `{{this.myModifier}}`.
+
+This has the added benefit of, with template strict mode, using plain functions in module-scope
+for helpers and modifiers.
+
+For Example:
+
+<!--
+  If you're reading this RFC non-rendered, ignore the "jsx" language tag.
+  "jsx" is incorrect, and isn't perfect, but "gjs" syntax highlighting
+  hasn't been added to GitHub's highlighting yet.
+-->
+
+```jsx
+const double = num => num * 2;
+const resizable = element => { /* ... */ };
+
+<template>
+  {{double 2}}
+  <div {{resizable}}></div>
+</template>
+```
+_Note: `<template>` is experimental syntax, and should not be be the focus of this example_
+
+Another aspect of which is exciting is that it becomes easier, in tests to grab
+the output of yielding components:
+
+```jsx
+test('...', async function (assert) {
+  let output = [];
+  const capture = (...args) => output = args;
+
+  await render(hbs`
+    <MyComponent as |yielded info and things|>
+       {{capture yielded info and things}}
+    </MyComponent>
+  `);
+
+  assert.equal(output[0], /* value of yielded */ );
+  assert.equal(output[1], /* value of info */ );
+  // ...
+})
+```
+
+A default modifier manager will be covered in a different RFC.
+
+## Detailed design
+
+The logic for choosing when to use a default manager is, at a high-level:
+
+```
+if (noExistingManagerFor(X)) {
+  return defaultManager;
+}
+```
+Where X is, for the purposes of this RFC, a Helper.
+
+_A Default Manager is not something that can be chosen by the user, but is baked in to the framework
+as a default so that a user doesn't have to build something to use a non-framework-specific variant
+of the three constructs: Helpers, Modifiers, and Components._
+
+Borrowing most of the implementation from ember-could-get-used-to-this with one change: the signature
+of the function may receive an options-like object as the last parameter if named arguments are specified.
+This aligns with the _syntax_ of helper invocation where named arguments may not appear before the last
+positional argument.
```

Comment:
This should describe the API from first principles (at the moment, it describes things as "what XYZ addon does, but with these differences")